atom/browser/api/atom_api_app.h

```diff
@@ -228,6 +232,10 @@ class App : public AtomBrowserClient::Delegate,
 
   base::FilePath app_path_;
 
+  util::Promise* gpuinfo_promise_;
```

Comment:
Can we explicitly initialize this to `nullptr`, if people do a truthy check on this in the future it wouldn't work without explicit initialization.

Reply:
I'm doing that in the constructor.